helmfile/50-applications.yaml

```diff
@@ -362,7 +362,7 @@ releases:
     app: opensearch-dashboards
     group: opensearch
   chart: ./upstream/opensearch-dashboards
-  version: 1.7.4
```

Comment:
Why is this `2.4.0` and not `2.5.0`?

Reply:
For some reason the code preview shows the wrong diff. But I ofc meant the other "side"

Reply:
https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch-dashboards/Chart.yaml

Because OpenSearch and OpenSearch-Dashboard don't have the same version number. Not super logical but!